packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BIContractActivationWindow.spec.jsx

```diff
@@ -106,7 +106,7 @@ describe('BIContractActivationWindow', () => {
 
     expect(fetchContractSynchronizationUrl).toHaveBeenCalled()
     expect(refreshContracts).toHaveBeenCalledTimes(1)
-    expect(InAppBrowser).toHaveBeenCalledWith(
+    return expect(InAppBrowser).toHaveBeenCalledWith(
```

Comment:
I'm not sure about the modification in this file. I was expecting to have return here https://github.com/cozy/cozy-libs/blob/5843aafb97484171f3c0a5de91162628ec4b4fd6/packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BIContractActivationWindow.spec.jsx#L66 here and here https://github.com/cozy/cozy-libs/blob/5843aafb97484171f3c0a5de91162628ec4b4fd6/packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BIContractActivationWindow.spec.jsx#L74-L75 

I'm not sure that the test pass correctly ATM.

Reply:
done